diagnostics-app/src/app/core/config/data-refresh-intervals.ts

```diff
@@ -1,5 +1,10 @@
-export const refreshIntervals = new Map<string, number> ([
-    ['memory', 1000],
-    ['storage', 60000],
-    ['cpu', 1000],
-])
+import { TelemetryInfoType } from '@common/message';
+
+export const refreshIntervals = {
+    telemetry: new Map<string, number> ([
+        [TelemetryInfoType.BATTERY, 1000],
+        [TelemetryInfoType.CPU, 1000],
+        [TelemetryInfoType.STATEFUL_PARTITION, 60000],
+        ['default', 10000],
```

Comment:
Use standalone variable for default value. e.g. `defaultTelemetryRefreshInterval`

Reply:
Done.